css/main.css

```diff
@@ -0,0 +1,344 @@
+html,
+body,
+.page {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    transition: all .6s cubic-bezier(.5, .2, .2, 1.1);
+    -webkit-transition: all .6s cubic-bezier(.5, .2, .2, 1.1);
+    -moz-transition: all .6s cubic-bezier(.5, .2, .2, 1.1);
+    -o-transition: all .6s cubic-bezier(.5, .2, .2, 1.1);
+    background-color: #101820FF color: #f2aa4cff;
+    overflow: hidden;
+}
+
+* {
+    font-family: 'open sans', 'lato', 'helvetica', sans-serif;
+}
+
+.page {
+    position: absolute;
+}
+
+#p1 {
+    left: 0;
+}
+
+#p2 {
```

Comment:
`id` selectors are specificity heavy and I generally avoid using them so as to prevent specificity issues. You may want to read more about it [here](https://css-tricks.com/specifics-on-css-specificity/)